index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <nav class="nav-bar">
+            <a class="logo decoration" href="#">My Logo</a>
+            <ul class="nav-items">
+                <li class="list-style"><a class="decoration" href="#">Portfolio</a></li>
+                <li class="list-style"><a class="decoration" href="#">About</a></li>
+                <li class="list-style"><a class="decoration" href="#">Contact</a></li>
```

Comment:
- [ ] As the previous reviewer suggested, because this milestone is just for the mobile version, please add the hamburger instead of this list.

![image](https://user-images.githubusercontent.com/43995830/158821003-cfa6a3f9-7d5c-437d-b33d-cc6840b2e907.png)